src/dev/flang/ast/Call.java

```diff
@@ -717,6 +725,17 @@ else if (_calledFeature == null) // nothing found, so flag error
   }
 
 
+  /**
+   * Does fo require an instance to be called?
```

Comment:
I needed a while to understand what this means, the comment just repeats the method name. This basically checks that neither fo._feature nor fo_outer are type features.